menas/src/main/scala/za/co/absa/enceladus/menas/utils/converters/StringToValidationKindConverter.scala

```diff
@@ -16,8 +16,8 @@
 package za.co.absa.enceladus.menas.utils.converters
 
 import org.springframework.core.convert.converter.Converter
-import za.co.absa.enceladus.menas.utils.enumerations.ValidationLevel
-import za.co.absa.enceladus.menas.utils.enumerations.ValidationLevel.ValidationLevel
+import za.co.absa.enceladus.utils.validation.ValidationLevel.ValidationLevel
```

Comment:
This seems a bit weird

Reply:
What exactly? In what way?

Reply:
ValidationLevel.ValidationLevel

Reply:
That's how it looks like when defining enumeration as a descendant of `Enumeration` and not as `case object`s.